rules/dispatch.go

```diff
@@ -38,59 +37,71 @@ func (r *DefaultRuleRenderer) WorkloadDispatchChains(
 	result := []*Chain{}
 	result = append(result,
 		// Assemble a from-workload and to-workload dispatch chain.
-		r.dispatchChains(
+		r.dispatchEndpointChains(
 			names,
-			epMarkMapper,
 			WorkloadFromEndpointPfx,
 			WorkloadToEndpointPfx,
 			ChainFromWorkloadDispatch,
 			ChainToWorkloadDispatch,
 			true,
 			true,
-			false,
 		)...,
 	)
 
-	if r.KubeIPVSSupportEnabled {
-		// In some scenario, e.g. packet goes to an kuberentes ipvs service ip. Traffic goes through input/output filter chain
-		// instead of forward filter chain. It is not feasible to match on an incoming workload interface with service ips.
-		// Assemble a set-endpoint-mark chain to set the endpoint mark matching on the incoming workload interface and
-		// a from-endpoint-mark chain to jump to a corresponding endpoint chain matching on its' endpoint mark.
-		result = append(result,
-			r.dispatchChains(
-				names,
-				epMarkMapper,
-				WorkloadSetEndPointMarkPfx,
-				WorkloadFromEndpointPfx,
-				ChainDispatchSetEndPointMark,
-				ChainDispatchFromEndPointMark,
-				false, // Non forwarded packet will pass through set endpoint mark chain.
-				true,
-				true,
-			)...,
-		)
-	}
 	return result
 }
 
+// In some scenario, e.g. packet goes to an kuberentes ipvs service ip. Traffic goes through input/output filter chain
+// instead of forward filter chain. It is not feasible to match on an incoming workload/host interface with service ips.
+// Assemble a set-endpoint-mark chain to set the endpoint mark matching on the incoming workload/host interface and
+// a from-endpoint-mark chain to jump to a corresponding endpoint chain matching on its' endpoint mark.
```

Comment:
typo: `s/its'/its/` (this part of English makes no sense whatsoever)

Reply:
Done.